src/App.vue

```diff
@@ -18,12 +19,24 @@ export default defineComponent({
   },
   setup() {
     const globalStore = useGlobalStore();
-    const loading = computed(() => globalStore.loading);
+    const loading1 = computed(() => globalStore.loading);
 
-    return {
-      loading,
-    };
+    provide('loading2', loading1)
```

Comment:
suggestion: you can name variable `loading`
`const loading = computed(() => globalStore.loading);`
`provide('loading', loading)`